agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -478,7 +479,8 @@ private static Configuration loadConfigurationFile(Path agentJarPath) {
       return getConfiguration(runtimeAttachedConfigurationContent, JsonOrigin.RUNTIME_ATTACHED);
     }
 
-    if (SdkVersionPrefixHolder.isRpIntegration()) {
+    // any manual RP integration will support loading user's applcationinsights.json
```

Comment:
(or do we need this, see comment in line below)
```suggestion
    // only RP integrations do not support loading applicationinsights.json
```